identity/backend/src/main/java/io/camunda/identity/usermanagement/service/UserService.java

```diff
@@ -98,13 +100,18 @@ public CamundaUser updateUser(final Long id, final CamundaUserWithPassword user)
       final UserDetails existingUserDetail =
           userDetailsManager.loadUserByUsername(existingUser.getUsername());
 
-      final UserDetails userDetails =
+      final var userBuilder =
           User.withUsername(existingUser.getUsername())
-              .password(user.getPassword())
-              .passwordEncoder(passwordEncoder::encode)
               .authorities(existingUserDetail.getAuthorities())
-              .disabled(!user.isEnabled())
-              .build();
+              .disabled(!user.isEnabled());
+
+      if (StringUtils.hasText(user.getPassword())) {
+        userBuilder
+            .password(user.getPassword())
+            .passwordEncoder(passwordEncoder::encode);
+      }
```

Comment:
It should not be needed to always provide a password. If left empty then the password will not change